src/main/scala/org/bitcoins/core/util/Base58.scala

```diff
@@ -65,10 +131,22 @@ trait Base58 extends BitcoinSLogger {
   def decode(input: String) : Seq[Byte] = {
     val zeroes = input.takeWhile(_ == '1').map(_ => 0:Byte).toArray
     val trim  = input.dropWhile(_ == '1').toList
-    val decoded = trim.foldLeft(BigInt(0))((a,b) =>a.*(BigInt(58L)).+(BigInt(base58Pairs(b))))
+    val decoded = trim.foldLeft(BigInt(0))((a,b) =>
+      a.*(BigInt(58L)).+(BigInt(base58Pairs(b))))
     if (trim.isEmpty) zeroes else zeroes ++ decoded.toByteArray.dropWhile(_ == 0)
   }
 
+  def isValid(base58 : String) : Boolean = {
+    val firstByte : Seq[Byte]= if (base58.isEmpty) List() else Seq(decode(base58).head)
+    val length = base58.length
+    val validFirstByteInHex = List("00", "05", "80", "6f", "c4", "ef")
```

Comment:
What are these numbers? These look awfully similar to some parameters I have in ChainParams